apps/pie-docs/src/assets/styles/components/_list.scss

```diff
@@ -43,22 +44,16 @@
     gap: f.spacing();
     flex-wrap: wrap;
 
-    .c-list-item {
+    & .c-list-item {
```

Comment:
Isn't this implied by the nesting?

Reply:
Yeh, we don't enforce this, but I prefer it for clarity (I find it helps with readability when scanning)